### PR TITLE
Rewrite changes status postbox

### PR DESCRIPTION
**Pull request by Moppa5**
#### What are the main changes in this PR?
Rewrite the changes status postbox on upkeep page. Also add the following things
- New template for date input
- Cleanup this widget's old code but some old upkeep code as well

### CODE REVIEW

File: src/modules/upkeep.php [hunk 1]
```diff
@@ -125,6 +117,29 @@ public static function init_upkeep_postboxes( Toolpage $page ) {
...
+      $changes_status->set_requirements(array( Requirements::CAN_BE_ANY_ENV => true ));
+      $changes_status->set_ajax_func(array( __CLASS__, 'fetch_backup_list_changes' ));
+      $changes_status->add_paragraph(
+        __(
+          'This tool can be used to run command <code>wp-backup-list-changes-since</code> 
+        which finds folder and file changes in backup data since the given date. For example if you have started to have issues on your site, you can
+        track down what folders or files have changed.  
+        
+         <p>Backups are stored for 30 days which is also the maximum since offset.  </p>',
+          'seravo'
```
Comment (Moppa5) [inline src/modules/upkeep.php:135]: Most likely should add this as a separate paragraph. Will fix with the possible other changes.

File: src/modules/upkeep.php [hunk 2]
```diff
@@ -125,6 +117,29 @@ public static function init_upkeep_postboxes( Toolpage $page ) {
...
+        )
+      );
+      $changes_status->set_build_form_func(array( __CLASS__, 'build_backup_list_changes' ));
+      $changes_status->set_button_text(__('Run', 'seravo'));
+      $changes_status->set_title_text(__('Click "run" to see changes', 'seravo'));
```
Comment (Moppa5) [inline src/modules/upkeep.php:140]: "run" => "Run" instead.

File: src/lib/postbox/component/template.php [hunk 3]
```diff
@@ -184,12 +184,25 @@ public static function checkbox_with_label( $label, $name, $checked = false ) {
...
+    return Component::from_raw($label . ' <input type="date" id="' . $id . '" name="' . $id . '" ' . (empty($min) ? '' : 'min="' . $min . '"') . (empty($max) ? '' : 'max="' . $max . '"') . '>');
+  }
+
   /* Add radio_button with the given details.
    * @param string $name Name for the radio button input.
    * @param string $value Value of the radio button.
    * @param string $content Content message for the radio button.
    * @param bool $checked Check the radiobutton.
    * @return \Seravo\Postbox\Component Button component.
+   * @return \Seravo\Postbox\Component
```
Comment (JoosuaKoskinen) [inline src/lib/postbox/component/template.php:205]: Some Rector thing?

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]
Looks good, only one weird comment! :+1

The old upkeep.css also messes with the new output wrapper and 'show more' is a bit miss aligned:
miss-align

File: src/lib/postbox/component/template.php (hunk 3 above)
Comment (Moppa5) [reply src/lib/postbox/component/template.php:205]: Probably yes.

**Review by JoosuaKoskinen — APPROVED** [review]

**Pull request merged**